src/webglimpse/timeline/timeline_events_row.ts

```diff
@@ -705,39 +710,87 @@ module Webglimpse {
                 '                                                                ',
                 '  void main( ) {                                                ',
                 '      gl_Position = vec4( ( -1.0 + 2.0*a_XyFrac ), 0.0, 1.0 );  ',
                 '      v_Color = a_Color;                                        ',
+                '      v_relativeXy = a_relativeXy;                                ',
+                '      v_fillPattern = a_fillPattern;                                ',                
                 '  }                                                             ',
                 '                                                                '
             );
 
-        var program = new Program( xyFrac_vColor_VERTSHADER, varyingColor_FRAGSHADER );
+
+         var fillPattern_FRAGSHADER = Webglimpse.concatLines(
+                '                                                                           ',
+				'  precision highp float;                                                   ',
+				'  varying vec4 v_Color;                                                    ',
+   				'  varying vec2 v_relativeXy;                                                ',
+   				'  varying float v_fillPattern;                                              ',                   
+				'  void main( ) {                                                           ',
+                '       if(v_fillPattern==0.){                                               ', // If fillPattern is solid, set the color and return
+				'           gl_FragColor = v_Color;                                         ',
+                '           return;                                                         ',
+                '       }                                                                   ',
+				'       float pi = 3.14159265359;                                           ',
+				'       float wave = sin(32.*2.*pi*(v_relativeXy.x+v_relativeXy.y*0.05));           ',
+				'       wave = (wave+1.)/2.;                                                ',
+				'       if(wave>0.95){                                                      ',
+				'           gl_FragColor = v_Color;                                         ',	
+				'       }else if(wave>0.4) {                                                ',
+				'           gl_FragColor = (wave + 0.3)*v_Color;                            ', // Add 0.3 to wave to reduce the dark outline of the stripes
+				'       } else {                                                            ',
+                '           gl_FragColor = vec4(1.);                                        ', // Make the background white
+                '       }	                                                                ',
+				'  }                                                                        ',
+				'                                                                           ' 
```

Comment:
Here's an example of what the above shader would look like (of course colors, stipple line with, and slant are adjustable):

![stripe-fill](https://cloud.githubusercontent.com/assets/1257858/19024308/1be6b792-88cf-11e6-854a-e5fb03f8dbad.png)